src/backend/sqlite/query.rs

```diff
@@ -49,4 +49,17 @@ impl QueryBuilder for SqliteQueryBuilder {
     ) {
         // Sqlite doesn't support sql recursive with query 'SEARCH' and 'CYCLE' options.
     }
+
+    fn prepare_select_distinct(
+        &self,
+        select_distinct: &SelectDistinct,
+        sql: &mut SqlWriter,
+        _collector: &mut dyn FnMut(Value),
+    ) {
+        match select_distinct {
+            SelectDistinct::All => write!(sql, "{}", "ALL").unwrap(),
+            SelectDistinct::Distinct => write!(sql, "{}", "DISTINCT").unwrap(),
+            _ => {}
+        }
+    }
```

Comment:
I guess we can safely remove this and fallback to the default implementation?

Reply:
Default implementation is blank. What do you mean?